src/loaders/svg/tvgSvgLoader.h

```diff
@@ -32,6 +32,8 @@ class SvgLoader : public LoadModule, public Task
     string svgPath = "";
     const char* content = nullptr;
     uint32_t size = 0;
+    AspectRatioAlign align = AspectRatioAlign::XMidYMid;
+    AspectRatioMeetOrSlice meetOrSlice = AspectRatioMeetOrSlice::Meet;
```

Comment:
Is this public member really necessary?
doc's align and meetOrslice are not changed, and the loaderData can be accessed wherever they are used.

Reply:
changed